web-client/src/presenter/sequences/closeVerifyEmailModalAndNavigateToPractitionerDetailSequence.js

```diff
@@ -0,0 +1,13 @@
+import { clearModalSequence } from './clearModalSequence';
+import { getUserContactEditCompleteAlertSuccessAction } from '../actions/getUserContactEditCompleteAlertSuccessAction';
+import { navigateToPractitionerDetailAction } from '../actions/navigateToPractitionerDetailAction';
+import { setAlertSuccessAction } from '../actions/setAlertSuccessAction';
+import { setSaveAlertsForNavigationAction } from '../actions/setSaveAlertsForNavigationAction';
+
+export const closeVerifyEmailModalAndNavigateToPractitionerDetailSequence = [
```

Comment:
No need to change anything, but might these compound sequences be broken up into two sequences for readability and reusability?

Reply:
I'm not sure I follow.  If anything, the sequence has a rather verbose name -- but the number of actions within is brief and easily read.